website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -29,17 +35,117 @@ saved state on startup if the state
 
 This bug is fixed in Consul versions 1.13.1 and newer.
 
-### gRPC TLS
-
-In prior Consul versions if HTTPS was enabled for the client API and exposed
-via `ports { https = NUMBER }` then the same TLS material was used to encrypt
-the gRPC port used for xDS. Now this is decoupled and activating TLS on the
-gRPC endpoint is controlled solely with the gRPC section of the new 
-[`tls` stanza](/docs/agent/config/config-files#tls-configuration-reference).
-
-If you have not yet switched to the new `tls` stanza and were NOT using HTTPS
-for the API then updating to Consul 1.13 will activate TLS for gRPC since the
-deprecated TLS settings are used as defaults.
+#### If using auto-encrypt or auto-config
+
+**Do not upgrade to Consul 1.13 yet** if using
+[auto-encrypt](/docs/agent/config/config-files#auto_encrypt) or
+[auto-config](/docs/agent/config/config-files#auto_config).
+
+In Consul 1.13, auto-encrypt and auto-config both cause Consul
+to require gRPC communication with Envoy proxies to now use TLS,
+but the Envoy proxies are unlikely to be configured to use gRPC.
+This will cause Envoy proxies (sidecars, gateways) to disconnect
+from the control plane (Consul agents).
+
+The underlying cause is the same as discussed in
+[gRPC TLS changes](#if-https-port-not-enabled-on-consul-agents-grpc-tls-changes).
+However, when using auto-encrypt or auto-config,
+the problem **cannot** currently be avoided by 
+[modifying the agent's TLS configuration](#modify-the-consul-agent-s-tls-configuration)
+because [`tls.defaults`](/docs/agent/config/config-files#tls_defaults)
+is automatically set.
```

Comment:
This isn't _entirely_ accurate, it's actually that we consider the auto-config/auto-encrypt cert alongside `tls.defaults` and `tls.grpc`, but it's the same outcome.

Reply:
Gotcha. So they don't set `tls.defaults`, but they are another source that TLS material is inherited from across all interfaces (just like `tls.defaults`)?

Reply:
Yep exactly. Sorry for the confusion, I definitely said or implied they set `tls.defaults` on Zoom yesterday!